src/pages/student/categories/CategoryDetail/index.js

```diff
@@ -95,42 +130,48 @@ function Subcategories() {
             </div>
           </p>
         </div>
-        <div id={style.quizlink}>
-          {category?.quizzes_count ? (
-            <a
-              className={style.linkToQuizzes}
-              href={`/categories/${category?.id}/quizzes`}
-            >
-              Check Available Quizzes &gt;&gt;
-            </a>
-          ) : (
-            ''
-          )}
-        </div>
       </div>
 
-      {categories === null ? (
+      { !data ? (
         <div className={style.loading}>
           <Spinner animation="border" role="status"></Spinner>
           <span className={style.loadingWord}>Loading</span>
         </div>
       ) : (
-        <div className={style.cardList}>{renderCatList()}</div>
-      )}
-
-      {categories && categories?.length <= 0 ? (
-        <div className={style.noResultsMessage}>
-          <p className={style.message}>NO RESULTS FOUND</p>
-        </div>
-      ) : (
-        <div className="pt-4">
-          <Pagination
-            page={page}
-            perPage={perPage}
-            totalItems={totalItems}
-            pageCount={lastPage}
-            onPageChange={onPageChange}
-          ></Pagination>
+        <div>
+          <Row className={style.cardList}>
+            {renderCatList()}
+          </Row>
+          {categories?.length <= 0 ? (
+            <div>{message('NO RESULTS FOUND')}</div>
```

Comment:
```suggestion
            <div>{noResultMessage ('NO RESULTS FOUND')}</div>
```

update the function name here as well after updating the previous one